frontend/src/pages/MentorDashboard.js

```diff
@@ -1,11 +1,40 @@
-import React from "react";
+import React, {Component} from 'react';
 
-class MentorDashboard extends React.Component {
-    render(){
-        return <div>
-            MENTOR DASHBOARD
-        </div>
+class MentorDashboard extends Component {
+     constructor(props) {
+    super(props);
+
+    this.state = {isLoading: true};
+  }
+    componentDidMount() {
+        let com = this
```

Comment:
comments would be helpful here

Reply:
This is for the user role based. It hits the api, if the user role is not valid, it will not get them to dashboard.
ex, if mentor is the user, it wont let them access the admin page. this applies for the student as well.